protocol/v1/qbft/instance/pre_prepare.go

```diff
@@ -27,6 +27,9 @@ func (i *Instance) PrePrepareMsgPipeline() pipelines.SignedMessagePipeline {
 				return fmt.Errorf("could not get proposal data: %w", err)
 			}
 			i.containersMap[specqbft.ProposalMsgType].AddMessage(signedMessage, proposalData.Data)
+
+			i.State().ProposalAcceptedForCurrentRound.Store(signedMessage)
+
```

Comment:
lets move it to be inside `UponPrePrepareMsg`

Reply:
@nivBlox Good idea, done